sdk/documentintelligence/ai-document-intelligence-rest/CHANGELOG.md

```diff
@@ -2,6 +2,27 @@
 
 # Release History
 
+## 1.1.0 (2025-05-08)
+
+### Features Added
+
+- Supports alternative cloud environments (Azure United States Government and Azure China Cloud). To use an alternative cloud environment, provide a value for the `scopes` field of `DocumentIntelligenceClientOptions#credentials` to configure the client to authenticate within a [Sovereign Cloud](https://learn.microsoft.com/entra/identity-platform/authentication-national-cloud).
```

Comment:
i may not quite understand the notes here, to be able to access for example Government clould - https://cognitiveservices.azure.us, i need to authenticate https://login.microsoftonline.us first?